packages/material-components-web/package.json

```diff
@@ -16,6 +16,7 @@
     "@material/animation": "^0.25.0",
     "@material/auto-init": "^0.29.0",
     "@material/base": "^0.29.0",
+    "@material/bottom-line": "^0.29.0",
```

Comment:
I think you might need to create a 0.0.0 version first? And then let lerna upgrade it for you when we release? Check with @kfranqueiro

Reply:
First commit of selection-control, for reference: https://github.com/material-components/material-components-web/commit/da9c7a99b2ab89c3b66ac4130a42fda548fdc01c